kolibri/plugins/device/assets/src/views/PinAuthenticationModal.vue

```diff
@@ -80,7 +80,7 @@
         context: 'Error message displayed when an incorrect PIN is input',
       },
       invalidPin: {
-        message: 'Please enter a 4-digit number',
+        message: 'Enter four numbers to set as your new PIN',
```

Comment:
Flagging that this change is the reason for the failing test, so this string needs to be updated in the test too.